### DISCUSSION TITLE
贡献者奖励记录

### DISCUSSION THREAD

**Discussion by deanplus** [lbc-team/UpchainDAO#14] [General]
因为小伙伴的贡献，我们[登链社区](https://learnblockchain.cn/) 才越来越好。

本贴用来记录社区小伙伴的贡献与奖励发放。

欢迎小伙伴们吐槽、建议和参与贡献，一起为 Web3 开发者成长构建更好的环境，更好的帮助开发者进入 Web3。

|  时间  |  姓名  |  钱包地址  |  贡献内容  |  奖励  |  发送状态  |
|  ----  |  ----  |  ----  |  ----  |  ----  |  ----  |
|  2024.05  |  yi     |  EVM: 0x15AD48F5225b3d6FCD5aB03650FAa39242faf7Df  |  登链社区新功能开发和维护  |    100U   |  [已发放](https://optimistic.etherscan.io/tx/0x703ec515af3ad0393927f3ae1114cb0b09d70eda26bcb9019c8a430a7d167b64)     |
|  2024.05  |  Ray    |  EVM:  0x32d4b70005516D1529277Da74482bb9d3d63eaFF  |  登链社区、Decert.Me 产品设计；OpenSpace 运营 |    700U   |  [已发放](https://optimistic.etherscan.io/tx/0x703ec515af3ad0393927f3ae1114cb0b09d70eda26bcb9019c8a430a7d167b64)     |
|  2024.05  |  dwong  |  EVM: 0x014EeCfA2E58d4975991F46026A2332561161912  |  DeCert.me 维护，添加教程（Rareskills soalna 教程翻译和校对） |    800U   |  [已发放](https://optimistic.etherscan.io/tx/0x703ec515af3ad0393927f3ae1114cb0b09d70eda26bcb9019c8a430a7d167b64)     |
|  2024.05  |  liangjies  |  EVM: 0x27e7c1006d361ca37116ec99dcacfa6db0b197f3  |  DeCert.me 开发（增强开放题评分功能（批注等）；显示未通过挑战）  |   500U    |  [已发放](https://optimistic.etherscan.io/tx/0x703ec515af3ad0393927f3ae1114cb0b09d70eda26bcb9019c8a430a7d167b64)     |
|  2024.05  |  Oaciqihz  | EVM: 0x3f3F05B76089200a49f05d99f88A2502A39e17DE|  DeCert.me 开发（增强开放题评分功能（批注等）；显示未通过挑战）  |   500U   |   [已发放](https://optimistic.etherscan.io/tx/0x703ec515af3ad0393927f3ae1114cb0b09d70eda26bcb9019c8a430a7d167b64)     |
|  2024.05  |  shooter  | EVM: 0xdCaD094D1c1d8fB816242A313644A5F6A44FD8f0 |             Rareskills soalna 教程校对  |    20U    |   [已发放](https://optimistic.etherscan.io/tx/0x703ec515af3ad0393927f3ae1114cb0b09d70eda26bcb9019c8a430a7d167b64)     |
|  2024.05  |  晓娜  |   EVM: 0xb190Ae1234B798cE5DADC545F2d4Df35eC40a4Ba   |            登链社区运营   |    1200 U      |   [已发放](https://optimistic.etherscan.io/tx/0x703ec515af3ad0393927f3ae1114cb0b09d70eda26bcb9019c8a430a7d167b64)     |
|  2024.05  |  Tiny熊  |  Starknet:   0x0335b1455da88ba0f65c91251711fd71473041d80064cafbaec9e0e83ee10abb    |          内容、对外合作等     |   1000 U     |   [已发放](https://starkscan.co/tx/0x076e729592797ed4540bf4761c0092e1fec656b2756d5a9e8463cf92f56cf25e)     |

往年贡献和激励发放记录：
- [登链社区时期贡献记录](https://github.com/orgs/lbc-team/discussions/1)
- [2024 年 4 月 回溯性奖励](https://github.com/orgs/lbc-team/discussions/13)

**Comment by xilibi2003** [comment]
6 月贡献发放记录

|  时间  |  姓名  |  钱包地址  |  贡献内容  |  奖励  |  发送状态  |
|  ----  |  ----  |  ----  |  ----  |  ----  |  ----  |
|  2024.06  |  yi     |  EVM: 0x15AD48F5225b3d6FCD5aB03650FAa39242faf7Df  |  登链社区新功能开发和维护  |   100U    |  [已发放](https://optimistic.etherscan.io/tx/0xf191fb751505341cd3d86f09ec2aaf824aee0762922bef2263fc3662144c953a)    |
|  2024.06  | 晓乔    |  EVM:  0x32d4b70005516D1529277Da74482bb9d3d63eaFF  |  登链社区、Decert.Me 产品设计；OpenSpace 运营 |   700U    |  [已发放](https://optimistic.etherscan.io/tx/0xf191fb751505341cd3d86f09ec2aaf824aee0762922bef2263fc3662144c953a)  |
|  2024.06  |  dwong  |  EVM: 0x014EeCfA2E58d4975991F46026A2332561161912  |  DeCert.me 维护; 登链社区图谱增加内容 |    821U   | [已发放](https://optimistic.etherscan.io/tx/0xf191fb751505341cd3d86f09ec2aaf824aee0762922bef2263fc3662144c953a)    |
|  2024.06  |  liangjies  |  EVM: 0x27e7c1006d361ca37116ec99dcacfa6db0b197f3  |  DeCert.me 开发（[挑战详情统计](https://github.com/decert-me/website/issues/418)、[挑战、教程页面筛选](https://github.com/decert-me/website/issues/421)等）  |   400U    |   [已发放](https://optimistic.etherscan.io/tx/0xf191fb751505341cd3d86f09ec2aaf824aee0762922bef2263fc3662144c953a)   |
|  2024.06  |  Oaciqihz  | EVM: 0x3f3F05B76089200a49f05d99f88A2502A39e17DE|  DeCert.me 开发（[挑战详情统计](https://github.com/decert-me/website/issues/418)、[挑战、教程页面筛选](https://github.com/decert-me/website/issues/421)等）  |  400U    | [已发放](https://optimistic.etherscan.io/tx/0xf191fb751505341cd3d86f09ec2aaf824aee0762922bef2263fc3662144c953a)     |
|  2024.06  |  晓娜  |   EVM: 0xb190Ae1234B798cE5DADC545F2d4Df35eC40a4Ba   |            登链社区运营   |     1200U      |   [已发放](https://optimistic.etherscan.io/tx/0xf191fb751505341cd3d86f09ec2aaf824aee0762922bef2263fc3662144c953a)    |
|  2024.06  |  Tiny熊  |  Starknet:   0x0335b1455da88ba0f65c91251711fd71473041d80064cafbaec9e0e83ee10abb    |          内容、对外合作等     |    1000U     |    [已发放](https://starkscan.co/tx/0x00f0dc641e644c3e80101f556da095e7e08cb607da5c53ae14e40d226696c1b5)    |
|  2024.06  |  847850277  | EVM: 0xF4C97a841e3259aB08B35926AB25EB65CF900aFf |             [Solana 文档翻译](https://github.com/lbc-team/developer-content/issues/1)  |    200 U    | [已发放](https://optimistic.etherscan.io/tx/0x375bf29031eda5c444dda21c74d2f5194d04cd532a29b01219ec39e2282a7358) |

**Comment by deanplus** [comment]
7 月贡献发放记录

|  时间  |  姓名  |  钱包地址  |  贡献内容  |  奖励  |  发送状态  |
|  ----  |  ----  |  ----  |  ----  |  ----  |  ----  |
|  2024.07  |  yi     |  EVM: 0x15AD48F5225b3d6FCD5aB03650FAa39242faf7Df  |  登链社区新功能开发和维护  |   100U    | [已发放](https://optimistic.etherscan.io/tx/0x7570afeaf3731458b3d55d9b15f7f768eadc860167ee7d32bbf447ead29ee21b)    |
|  2024.07  | 晓乔    |  EVM:  0x32d4b70005516D1529277Da74482bb9d3d63eaFF  |  登链社区、Decert.Me 产品设计；OpenSpace 运营 |   297.75 U    | [已发放](https://optimistic.etherscan.io/tx/0x7570afeaf3731458b3d55d9b15f7f768eadc860167ee7d32bbf447ead29ee21b)  |
|  2024.07  |  dwong  |  EVM: 0x014EeCfA2E58d4975991F46026A2332561161912  |  DeCert.me 维护； Solidity 101 挑战 |    810U   |  [已发放](https://optimistic.etherscan.io/tx/0x7570afeaf3731458b3d55d9b15f7f768eadc860167ee7d32bbf447ead29ee21b)   |
|  2024.07  |  liangjies  |  EVM: 0x27e7c1006d361ca37116ec99dcacfa6db0b197f3  |  DeCert.me 开发和维护|   500U    | [已发放](https://optimistic.etherscan.io/tx/0x7570afeaf3731458b3d55d9b15f7f768eadc860167ee7d32bbf447ead29ee21b)     |
|  2024.07  |  Oaciqihz  | EVM: 0x3f3F05B76089200a49f05d99f88A2502A39e17DE|  DeCert.me 开发和维护 |  500U    |  [已发放](https://optimistic.etherscan.io/tx/0x7570afeaf3731458b3d55d9b15f7f768eadc860167ee7d32bbf447ead29ee21b)     |
|  2024.07  |  晓娜  |   EVM: 0xb190Ae1234B798cE5DADC545F2d4Df35eC40a4Ba   |  登链社区运营   |     1200U      |  [已发放](https://optimistic.etherscan.io/tx/0x7570afeaf3731458b3d55d9b15f7f768eadc860167ee7d32bbf447ead29ee21b)    |
|  2024.07  |  Tiny熊  |  Starknet:   0x0335b1455da88ba0f65c91251711fd71473041d80064cafbaec9e0e83ee10abb    |          内容、对外合作等     |    1000U     |  [已发放](https://starkscan.co/tx/0x02ded7019f9ee2475afd008ddb33f5e42bd17e888f24f0178decc2025f2173dc)    |
|  2024.07  |  mutourend  | EVM: 0xe2aD65DFFf579032Ef7c12d8be22Ed27F638eB4B |            ZKP 图谱 |    150 U    | [已发放](https://optimistic.etherscan.io/tx/0x7570afeaf3731458b3d55d9b15f7f768eadc860167ee7d32bbf447ead29ee21b)  |

**Comment by deanplus** [comment]
8 月贡献发放记录

|  时间  |  姓名  |  钱包地址  |  贡献内容  |  奖励  |  发送状态  |
|  ----  |  ----  |  ----  |  ----  |  ----  |  ----  |
|  2024.08  |  yi          |  EVM: 0x15AD48F5225b3d6FCD5aB03650FAa39242faf7Df  |  登链社区新功能开发和维护  |   100U    | [已发放](https://optimistic.etherscan.io/tx/0x25dd1d9dbc71712ebaafcaae7cea575be410c90cf98762d15415bb43d80109c6)   |
|  2024.08  |  dwong       |  EVM: 0x014EeCfA2E58d4975991F46026A2332561161912  |  DeCert.me 维护； Solidity 102 挑战 |    721.2U   |  [已发放](https://optimistic.etherscan.io/tx/0x25dd1d9dbc71712ebaafcaae7cea575be410c90cf98762d15415bb43d80109c6)   |
|  2024.08  |  liangjies   |  EVM: 0x27e7c1006d361ca37116ec99dcacfa6db0b197f3  |  DeCert.me 开发和维护|   300U    | [已发放](https://optimistic.etherscan.io/tx/0x25dd1d9dbc71712ebaafcaae7cea575be410c90cf98762d15415bb43d80109c6)     |
|  2024.08  |  Oaciqihz    | EVM: 0x3f3F05B76089200a49f05d99f88A2502A39e17DE|  DeCert.me 开发和维护 |  300U    |  [已发放](https://optimistic.etherscan.io/tx/0x25dd1d9dbc71712ebaafcaae7cea575be410c90cf98762d15415bb43d80109c6)     |
|  2024.08  |  晓娜         |   EVM: 0xb190Ae1234B798cE5DADC545F2d4Df35eC40a4Ba   |  登链社区运营   |     1200U      |  [已发放](https://optimistic.etherscan.io/tx/0x25dd1d9dbc71712ebaafcaae7cea575be410c90cf98762d15415bb43d80109c6)    |
|  2024.08  |  Tiny熊      |  Starknet:   0x0335b1455da88ba0f65c91251711fd71473041d80064cafbaec9e0e83ee10abb    |          内容、对外合作等     |    1000U     |  [已发放](https://starkscan.co/tx/0x07790c6d3af258b2ccdb6e41f3dc633719fe9977fa3bd352d4991b87d42a9056)   |
|  2024.08  |  EchoEcho    |  EVM:   0x3A8492819b0C9AB5695D447cbA2532b879d25900    |          [OpenSpace-Hackathon-Starknet 黑客松](https://github.com/lbc-team/OpenSpace-Hackathon) 第2名    |    200 U     |  [已发放](https://optimistic.etherscan.io/tx/0x25dd1d9dbc71712ebaafcaae7cea575be410c90cf98762d15415bb43d80109c6)    |
|  2024.08  |  RedPacket   |  EVM:   0x758e4e534afbb044dea64ab07e5d783fec7e1541    |          [OpenSpace-Hackathon-Starknet 黑客松](https://github.com/lbc-team/OpenSpace-Hackathon) 第2名    |    200 U     |  [已发放](https://optimistic.etherscan.io/tx/0x25dd1d9dbc71712ebaafcaae7cea575be410c90cf98762d15415bb43d80109c6)    |
|  2024.08  |  wukong      |  EVM:   0x7E480e349a31F71f59B57cb30Bd850b4D643D617    |          [OpenSpace-Hackathon-Starknet 黑客松](https://github.com/lbc-team/OpenSpace-Hackathon)  第3名      |    100 U     |  [已发放](https://optimistic.etherscan.io/tx/0x25dd1d9dbc71712ebaafcaae7cea575be410c90cf98762d15415bb43d80109c6)    |
|  2024.08  |  MDLedger    |  EVM:   0x65034a9364DF72534d98Acb96658450f9254ff59    |          [OpenSpace-Hackathon-Starknet 黑客松](https://github.com/lbc-team/OpenSpace-Hackathon) 第3名    |    100 U     |  [已发放](https://optimistic.etherscan.io/tx/0x25dd1d9dbc71712ebaafcaae7cea575be410c90cf98762d15415bb43d80109c6)    |

**Comment by deanplus** [comment]
9 月贡献发放记录

|  时间  |  姓名  |  钱包地址  |  贡献内容  |  奖励  |  发送状态  |
|  ----  |  ----  |  ----  |  ----  |  ----  |  ----  |
|  2024.09  |  yi          |  EVM: 0x15AD48F5225b3d6FCD5aB03650FAa39242faf7Df  |  登链社区新功能开发和维护  |   100U    | [已发放](https://optimistic.etherscan.io/tx/0xc451e285784caf2d2b55e863c027d40d262b54c6b12e6187a34015f48f2615e5)   |
|  2024.09  |  dwong       |  EVM: 0x014EeCfA2E58d4975991F46026A2332561161912  |  DeCert.me 维护； Solidity 102 挑战 |    845U   | [已发放](https://optimistic.etherscan.io/tx/0xc451e285784caf2d2b55e863c027d40d262b54c6b12e6187a34015f48f2615e5)    |
|  2024.09  |  liangjies   |  EVM: 0x27e7c1006d361ca37116ec99dcacfa6db0b197f3  |  DeCert.me 开发和维护|   50U    |  [已发放](https://optimistic.etherscan.io/tx/0xc451e285784caf2d2b55e863c027d40d262b54c6b12e6187a34015f48f2615e5)    |
|  2024.09  |  Oaciqihz    | EVM: 0x3f3F05B76089200a49f05d99f88A2502A39e17DE|  DeCert.me 开发和维护 |  50U    |  [已发放](https://optimistic.etherscan.io/tx/0xc451e285784caf2d2b55e863c027d40d262b54c6b12e6187a34015f48f2615e5)    |
|  2024.09  |  晓娜         |   EVM: 0xb190Ae1234B798cE5DADC545F2d4Df35eC40a4Ba   |  登链社区运营   |     1200U      |  [已发放](https://optimistic.etherscan.io/tx/0xc451e285784caf2d2b55e863c027d40d262b54c6b12e6187a34015f48f2615e5)    |
|  2024.09  |  popo         |   EVM: 0x4A2592b3dd6522C21101938d565dF9C08E867a56   |  登链社区运营   |     344.8 U      |  [已发放](https://optimistic.etherscan.io/tx/0xc451e285784caf2d2b55e863c027d40d262b54c6b12e6187a34015f48f2615e5)    |
|  2024.09  |  Tiny熊      |  Starknet:   0x0335b1455da88ba0f65c91251711fd71473041d80064cafbaec9e0e83ee10abb    |          内容、对外合作等     |    1000U     |  [已发放](https://starkscan.co/tx/0x05072b945ad196471d6f685d6340b0cae117c55e3341980cff95207e096ecc1c)   |

**Comment by deanplus** [comment]
11 月贡献发放记录

|  时间  |  姓名  |  钱包地址  |  贡献内容  |  奖励  |  发送状态  |
|  ----  |  ----  |  ----  |  ----  |  ----  |  ----  |
|  2024.11  |  yi          |  EVM: 0x15AD48F5225b3d6FCD5aB03650FAa39242faf7Df  |  登链社区新功能开发和维护  |   100U    | [已发放](https://optimistic.etherscan.io/tx/0xf69815c65e023e96273f915664aba2882685340bd61caba5da5261f7743eae18) |
|  2024.11  |  dwong       |  EVM: 0x014EeCfA2E58d4975991F46026A2332561161912  |  DeCert.me 维护；  foundry 中文文档更新；Viem 文档翻译成中文  |    800U   | [已发放](https://optimistic.etherscan.io/tx/0xf69815c65e023e96273f915664aba2882685340bd61caba5da5261f7743eae18)  |
|  2024.11  |  liangjies   |  EVM: 0x27e7c1006d361ca37116ec99dcacfa6db0b197f3  |  DeCert.me 开发和维护|   50U    |  [已发放](https://optimistic.etherscan.io/tx/0xf69815c65e023e96273f915664aba2882685340bd61caba5da5261f7743eae18)  |
|  2024.11  |  Oaciqihz    | EVM: 0x3f3F05B76089200a49f05d99f88A2502A39e17DE|  DeCert.me 开发和维护 |  50U    |  [已发放](https://optimistic.etherscan.io/tx/0xf69815c65e023e96273f915664aba2882685340bd61caba5da5261f7743eae18)  |
|  2024.11  |  popo         |   EVM: 0x4A2592b3dd6522C21101938d565dF9C08E867a56   |  登链社区运营   |    400U        |  [已发放](https://optimistic.etherscan.io/tx/0xf69815c65e023e96273f915664aba2882685340bd61caba5da5261f7743eae18)  |
|  2024.11  |  晓娜         |   EVM: 0xb190Ae1234B798cE5DADC545F2d4Df35eC40a4Ba   |  登链社区运营   |     1200U      |   [已发放](https://optimistic.etherscan.io/tx/0xf69815c65e023e96273f915664aba2882685340bd61caba5da5261f7743eae18) |
|  2024.11  |  Tiny熊      |  Starknet:   0x0335b1455da88ba0f65c91251711fd71473041d80064cafbaec9e0e83ee10abb    |          内容、对外合作等     |    1000U     |    [已发放](https://voyager.online/tx/0x28ce3cc22babdb043f4fb37e6b2c252cb6f8afc61b3029042835a6227010dde) |

**Comment by deanplus** [comment]
12 月贡献发放记录

|  时间  |  姓名  |  钱包地址  |  贡献内容  |  奖励  |  发送状态  |
|  ----  |  ----  |  ----  |  ----  |  ----  |  ----  |
|  2024.12  |  yi          |  EVM: 0x15AD48F5225b3d6FCD5aB03650FAa39242faf7Df  |  登链社区新功能开发和维护  |   100U    | [已发放](https://optimistic.etherscan.io/tx/0x9852214061ae24fffd3c9b320446f09fcd9978cd0d232be7205e7b325ec26979) |
|  2024.12  |  dwong       |  EVM: 0x014EeCfA2E58d4975991F46026A2332561161912  |  DeCert.me 维护； “技术英语一起学”功能开发   |    764.83U   | [已发放](https://optimistic.etherscan.io/tx/0x9852214061ae24fffd3c9b320446f09fcd9978cd0d232be7205e7b325ec26979)  |
|  2024.12  |  popo         |   EVM: 0x4A2592b3dd6522C21101938d565dF9C08E867a56   |  登链社区运营   |    400U        |  [已发放](https://optimistic.etherscan.io/tx/0x9852214061ae24fffd3c9b320446f09fcd9978cd0d232be7205e7b325ec26979)  |
|  2024.12  |  晓娜         |   EVM: 0xb190Ae1234B798cE5DADC545F2d4Df35eC40a4Ba   |  登链社区运营   |     1200U      |   [已发放](https://optimistic.etherscan.io/tx/0x9852214061ae24fffd3c9b320446f09fcd9978cd0d232be7205e7b325ec26979) |
|  2024.12  |  Tiny熊      |  Starknet:   0x0335b1455da88ba0f65c91251711fd71473041d80064cafbaec9e0e83ee10abb    |          内容、对外合作等     |    1000U     |    [已发放](https://voyager.online/tx/0x58eea26a60ec62480c66d9e1e9f568b1255c8e65372608b49d5a8ccae26b0ee) |

**Comment by deanplus** [comment]
2025.01 月贡献发放记录

|  时间  |  姓名  |  钱包地址  |  贡献内容  |  奖励  |  发送状态  |
|  ----  |  ----  |  ----  |  ----  |  ----  |  ----  |
|  2025.01  |  yi          |  EVM: 0x15AD48F5225b3d6FCD5aB03650FAa39242faf7Df  |  登链社区新功能开发和维护  |   100U    | [发放记录](https://optimistic.etherscan.io/tx/0x50f5250beb04f3e5cdefd74267b0e69e164733522a5b6af19bde897688eb9e6c) |
|  2025.01  |  dwong       |  EVM: 0x014EeCfA2E58d4975991F46026A2332561161912  |  DeCert.me 维护； “技术英语一起学”功能开发   |    830U   | [发放记录](https://optimistic.etherscan.io/tx/0x50f5250beb04f3e5cdefd74267b0e69e164733522a5b6af19bde897688eb9e6c)  |
|  2025.01  |  popo         |   EVM: 0x4A2592b3dd6522C21101938d565dF9C08E867a56   |  登链社区运营   |    350U        |  [发放记录](https://optimistic.etherscan.io/tx/0x50f5250beb04f3e5cdefd74267b0e69e164733522a5b6af19bde897688eb9e6c)  |
|  2025.01  |  晓娜         |   EVM: 0xb190Ae1234B798cE5DADC545F2d4Df35eC40a4Ba   |  登链社区运营   |     1200U      |   [发放记录](https://optimistic.etherscan.io/tx/0x50f5250beb04f3e5cdefd74267b0e69e164733522a5b6af19bde897688eb9e6c) |
|  2025.01  |  Tiny熊      |  Starknet:   0x0335b1455da88ba0f65c91251711fd71473041d80064cafbaec9e0e83ee10abb    |          内容、对外合作等     |    1000U     |    [未发放](https://voyager.online/tx/0x4ebb2845afc1d6ffbc7348069b2708eb15c97095034dd5eba555a8e2a19d4d6) |
